data/heightmap_metadata.yaml

```diff
@@ -0,0 +1,7 @@
+image: heightmap.pgm
+resolution: 0.025
+origin: [0.0, 0.0, 0.0]
+occupied_thresh: 1
+free_thresh: 0
+negate: 0
+mode: scale
```

Comment:
```suggestion

```

Mode is optional, I think we should use the default option, unless we have a very clear reason to.
I think scale is mostly there to have a gradient between occupied and unoccupied, we don't need that in this case, right?